packages/sui-studio/package.json

```diff
@@ -27,6 +27,7 @@
     "@s-ui/helpers": "1",
     "@s-ui/mono": "1",
     "@s-ui/react-context": "1",
+    "@s-ui/react-router": "^1.0.0-beta.10",
```

Comment:
Using the beta with caret to get new betas if available. We will change to major as soon as it's stable.